temporian/core/operators/drop_index.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""DropIndex operator."""
+from typing import List, Optional, Union
+
+from temporian.core import operator_lib
+from temporian.core.data.event import Event
+from temporian.core.data.feature import Feature
+from temporian.core.data.sampling import Sampling
+from temporian.core.operators.base import Operator
+from temporian.proto import core_pb2 as pb
+
+
+class DropIndexOperator(Operator):
+    """DropIndex operator."""
+
+    def __init__(
+        self,
+        event: Event,
+        labels: Optional[Union[List[str], str]] = None,
+        keep: bool = True,
+    ) -> None:
+        super().__init__()
+
+        # process labels
+        labels = self._process_labels(event, labels)
+
+        # input event
+        self.add_input("event", event)
+
+        # attributes
+        self.add_attribute("labels", labels)
```

Comment:
add_attribute does not support None type. Is the None type necessary here?

Note: If the None type means "dropping all the index", I would have this logic in the api function (instead of the api class).